src/components/application_manager/test/commands/command_request_impl_test.cc

```diff
@@ -185,6 +185,7 @@ TEST_F(CommandRequestImplTest, OnTimeOut_StateAwaitingHMIResponse_SUCCESS) {
 
   command->onTimeOut();
 
+  EXPECT_TRUE((*dummy_msg)[am::strings::msg_params][am::strings::info].asString().empty());
```

Comment:
@pold500 Please add some comment why here info should be empty or change this check